easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.2-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'GetOrganelle'
+version = '1.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Kinggerm/GetOrganelle'
+description = """This toolkit assemblies organelle genome from genomic skimming data."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/Kinggerm/GetOrganelle/archive/%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['54923ca971d68620aaf036a324b554084b00f7c209b0b3a234f815c82ee6d079']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('Bandage', '0.8.1', '_Centos', SYSTEM),
```

Comment:
@deniskristak Please use `True` as last value here, rather than `SYSTEM` (that should only be used to define `toolchain`)